events.go

```diff
@@ -159,57 +159,57 @@ func (s *Session) handleNodeEvent(frames []frame) {
 		}
 	}
 
-	for _, f := range events {
+	for _, event := range events {
```

Comment:
Can we remain the orig name? It would be easier to maintain.

Reply:
I understand that but it also modifies lines that would not be otherwise modified and thus makes it harder to review / maintain in the future.

Reply:
brought it back